src/version/sagas/contrastModeSagas.js

```diff
@@ -0,0 +1,13 @@
+import { takeEvery, put } from 'redux-saga/effects';
+import { get } from "lodash";
+
+import { CONTRAST_MODE_ACTION, contrastModeAction } from "../actions/contrastModeAction";
+
+export default takeEvery(CONTRAST_MODE_ACTION.REQUEST, function*(action) {
```

Comment:
you don't need a saga for that, it's not a side effect and it's not asynchronous. A reducer is enough and less complex.

Reply:
Saga was removed